src/embed.ts

```diff
@@ -20,8 +20,8 @@ export const vega = vegaImport;
 export let vegaLite = vegaLiteImport;
 
 // For backwards compatibility with Vega-Lite before v4.
-const w = window as any;
-if (vegaLite === undefined && w['vl'] && w['vl'].compile) {
+const w = (typeof window !== 'undefined' ? window : undefined) as any;
+if (vegaLite === undefined && w !== undefined && w['vl'] && w['vl'].compile) {
```

Comment:
I think we can simplify this, no? 

```suggestion
if (vegaLite === undefined && w.?['vl'].?compile) {
```